backend/tracim_backend/config.py

```diff
@@ -542,6 +544,20 @@ def _load_search_config(self):
             self.get_raw_config("search.elasticsearch.request_timeout", "60")
         )
 
+    def _load_collaborative_document_edition_config(self):
+        self.COLLABORATIVE_DOCUMENT_EDITION__ACTIVATED = asbool(
+            self.get_raw_config("collaborative_document_edition.activated", "false")
+        )
+        self.COLLABORATIVE_DOCUMENT_EDITION__SOFTWARE = self.get_raw_config(
+            "collaborative_document_edition.software"
+        )
+        self.COLLABORATIVE_DOCUMENT_EDITION__COLLABORA__BASE_URL = self.get_raw_config(
+            "collaborative_document_edition.collabora.base_url"
```

Comment:
why use collabora in the naming? The url is a standard (same for office365, libreoffice or collabora), no?

Reply:
not really. anything outside wopi scope is not standard (and we do not  use wopi endpoint, we provide them). we do a specific check with this url to get supported content_type in the specifc case of collabora/libreoffice